docs/releases/5.2.1.txt

```diff
@@ -51,3 +51,7 @@ Bugfixes
 
 * Fixed a regression in Django 5.2 that caused a crash when serializing email
   alternatives or attachments due to named tuple mismatches (:ticket:`36309`).
+
+* Fixed a regression in Django 5.2 that caused a redundant ``unique_together``
```

Comment:
I would avoid calling it a regression to reinforce that it's a bug in a new feature.

Reply:
Good point @jacobtylerwalls, thank you! I will push a tweak. How do you feel about:
```suggestion
* Fixed a bug in Django 5.2 that caused a redundant ``unique_together``
```

Reply:
Perfect!